contribute/developer-guide.md

```diff
@@ -224,57 +224,73 @@ make test-go-integration-postgres
 
 ### Run end-to-end tests
 
-Grafana uses [Cypress](https://www.cypress.io/) to end-to-end test core features. Core plugins use [Playwright](https://playwright.dev/) to run automated end-to-end tests. You can find more information on how to add end-to-end tests to your core plugin [in our end-to-end testing style guide](./style-guides/e2e-plugins.md)
+Grafana uses [Playwright](https://playwright.dev/) to run automated end-to-end tests. You can find more information on how to add end-to-end tests to your core plugin [in our end-to-end testing style guide](./style-guides/e2e-playwright.md#playwright-for-plugins)
```

Comment:
These docs are great and very in depth!

I think in this main developer-guide.md, we should focus on just the core flow and leave the rest to the seperate guide?

Maybe in `developer-guide.md` we have just:
 - Grafana uses Playwright, here's out full guide
 - run `yarn playwright install chromium` before to install the browser binary
 - run tests with `yarn e2e:playwright [optional path to test file]`
 - here's how you can download test results from a github action and view the report locally

thoughts? @ashharrison90